stringtie.wdl

```diff
@@ -33,6 +33,7 @@ task Stringtie {
 
         Int threads = 1
         String memory = "10G"
```

Comment:
Does it need that much? I have checked a 400ish MB BAM file and usage seems to be similar. But there is also a 1.3GB gtf involved.
Hmm. This is undecided.
Can we put memory at a static 2G for now? From MaxVMSize seems to be 5x the MaxRSS number. So I think we can safely lower memory by 5X. 
If we run into issues we can look at setting it dynamically.